vm/hash_test.go

```diff
@@ -623,47 +746,17 @@ func TestHashEqualMethodFail(t *testing.T) {
 	}
 }
 
-func TestHashDeleteMethod(t *testing.T) {
+func TestHashFetchMethod(t *testing.T) {
 	tests := []struct {
 		input    string
-		expected interface{}
+		expected string
 	}{
 		{`
-		h = { a: 1, b: "Hello", c: true }.delete("a")
-		h["a"]
-		`, nil},
-		{`
-		h = { a: 1, b: "Hello", c: true }.delete("a")
-		h["b"]
-		`, "Hello"},
-		{`
-		h = { a: 1, b: "Hello", c: true }.delete("a")
-		h["c"]
-		`, true},
+			{ spaghetti: "eat" }.fetch("pizza", "not eat")
```

Comment:
Why don't you test the successfully fetching condition?

Reply:
You're right, that was an omission :smile: